### PR TITLE
gh/workflows: Enable kube-proxy in some of DP conformance tests

### PR DESCRIPTION
**Pull request by brb**
* `4.19` - kube-proxy only.
 * `5.4` - mixed (kube-proxy and KPR).
 * `>= 5.10` - KPR only.

The passing CI run - https://github.com/cilium/cilium/actions/runs/3435933273.

Related https://github.com/cilium/cilium/issues/22116.

### CODE REVIEW

**Comment by brb** [comment]
Blocked by https://github.com/cilium/little-vm-helper-images/pull/31.

File: .github/workflows/conformance-datapath.yaml [hunk 1]
```diff
@@ -197,7 +215,7 @@ jobs:
             --helm-set=hubble.relay.image.tag=${{ steps.vars.outputs.sha }} \
             --rollback=false \
             --config monitor-aggregation=none \
-            --nodes-without-cilium=kind-worker3 --kube-proxy-replacement=strict --helm-set-string="k8sServiceHost=kind-control-plane,k8sServicePort=6443"
```
Comment (joestringer) [inline .github/workflows/conformance-datapath.yaml:200]: What happened to the k8s service configuration?

**Comment by joestringer** [comment]
Incidentally, does "mixed" work? Someone was asking on Slack this week about this and I assumed that we didn't validate kube-proxy + kubeProxyReplacement=strict together because they can conflict with one another.

File: .github/workflows/conformance-datapath.yaml (hunk 1 above)
Comment (brb) [reply .github/workflows/conformance-datapath.yaml:200]: [Autodetection](https://github.com/cilium/cilium-cli/pull/1175) :sunglasses:

**Comment by brb** [comment]
> Incidentally, does "mixed" work?

@joestringer Yep. The current 4.19 Jenkins pipeline runs in the mixed mode.

**Review by joestringer — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Comment by brb** [comment]
It's a fairly simple PR, so 2 ACK's is enough. Marking as ready to merge.

**Pull request merged**